### PR TITLE
Api Cleanup

### PR DESCRIPTION
**Pull request by lowtorola**
- Changed API/Auth to be exported functions in separate files.
- Accidentally changed ~5000 lines by running format (oopsie).

:rocket:

### CODE REVIEW

**Comment by lowtorola** [comment]
halp @acrantel my lint seems to be borked 😢

**Review by acrantel — CHANGES_REQUESTED** [review]
thanks for doing this!

what command did you run that formatted all the autogenerated files? if it was `npm run format` or `lint` then we'll need to update the ignores in `eslintrc.js` and `.prettierignore` . it should be fairly straightforward, just put the path to the autogenerated folder in the ignore lists

we shouldn't commit the format changes to the autogenerated files, here is a stackoverflow solution for how to remove them from your pr: https://stackoverflow.com/a/39460058

File: frontend2/.gitignore [hunk 1]
```diff
@@ -11,6 +11,9 @@
 # production
 /build
 
+# auto-generated files
+/src/utils/types
```
Comment (acrantel) [inline frontend2/.gitignore:15]: we shouldn't add this change, we want these types to stay in our git repo

Comment (lowtorola) [reply frontend2/.gitignore:15]: Oh whoops that's right this should be in `.prettierignore` 🤦

**Review by acrantel — APPROVED** [review]
lgtm! 🎉 :shipit:

**Pull request merged**